### DISCUSSION TITLE
Other Emulators

### DISCUSSION THREAD

**Discussion by gladel** [eduardofilo/RG350_adam_image#92] [Ideas & Requests]
Notice OPKs for Magnavox Odyssey and Atari ST included in Adam image. Also read Phillips Videopac BIOS mentioned in the Wiki. Will they be activated in the future?

**Comment by eduardofilo** [comment]
Well, there are many more OPKs than those configured in SimpleMenu because the cores occupy little space and thus users have the option to expand the functionality of the image.
I suppose that with time I will "activate" some of those cores in SimpleMenu, but there is no clear plan. One of the reasons that makes "activation" difficult is that for some of them there are no graphical resources in some of the four SimpleMenu themes installed in the image.

**Comment by gladel** [comment]
[Epic Noir](https://github.com/jad3948/Epic-Noir-RG350) theme has more graphical resources. Maybe use it as default if possible.

**Comment by SuperBadger77** [comment]
If it helps... I've edited Epic Noir to support BBC Micro - can send you the files :-)

**Comment by eduardofilo** [comment]
The problem is that I have to incorporate the systems into the four themes.
@SuperBadger77 of course, send me the files, please.

**Comment by gladel** [comment]
@eduardofilo I understand your intentions. Thank you.

**Comment by gladel** [comment]
@eduardofilo I'm not persistent, but I saw graphical assets on all included themes for Neo Geo Pocket Color and WonderSwan Color. Will you have any intention to add them in the future or just let users decide?

**Comment by eduardofilo** [comment]
Well, with those two systems the reason for not having included them was rather that it seemed better to unify the B&W and color versions to simplify. That is, it is a matter of taste.
